src/client/datascience/jupyter/kernels/helpers.ts

```diff
@@ -905,7 +905,7 @@ export async function sendTelemetryForPythonKernelExecutable(
 
 export async function executeSilently(session: IJupyterSession, code: string): Promise<nbformat.IOutput[]> {
     traceInfo(
-        `Executing (status ${session.status}) silently Code = ${code.substring(0, 100).splitLines().join('\\n')}`
+        `Executing silently Code (${session.status}) = ${code.substring(0, 100).splitLines().join('\\n')}`
```

Comment:
Changed this as when I was looking at th emessages i saw
`Executing (status idle) silently code: xyz`
`Executing silently code (completed) : xyz`

Both looked off to me, just changed so that the status is in brackets & its easy to identify the two easily and align them visually (i feel its easier to read)